### PR TITLE
update documentation for signing when using cosign

### PR DESCRIPTION
**Pull request by cpanato**
Update documentation for the signing part using cosign using the updated flags

### CODE REVIEW

File: www/docs/customization/docker_sign.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ docker_signs:
 
     # Command line templateable arguments for the command
     #
-    # defaults to `["sign", "-key=cosign.key", "${artifact}"]`
-    args: ["sign", "-key=cosign.key", "-upload=false", "${artifact}"]
+    # defaults to `["sign", "--key=cosign.key", "${artifact}"]`
+    args: ["sign", "--key=cosign.key", "--upload=false", "${artifact}"]
```
Comment (caarlos0) [inline www/docs/customization/docker_sign.md:30]: we should probably change this in code as well, I think the docker signs defaults to single dash , will do in another pr

**Comment by caarlos0** [comment]
Thanks!

**Comment by cpanato** [comment]
I can do that @caarlos0
